File: C++/Iniciante/1019.cpp
#include <stdio.h>
#include <iostream>

int main() {

  int segundos, horas, minutos;
  std::cin >> segundos;

  horas = segundos / 3600;
  segundos %= 3600;

  minutos = segundos / 60;
  segundos %= 60;

  printf("%d:%d:%d\n", horas, minutos, segundos);
    
}

File: GO/Iniciante/1019.go
package main

import (
	"fmt"
)

func main() {

  var segundos int
  fmt.Scan(&segundos)

  horas := segundos / 3600
  segundos %= 3600
  
  minutos := segundos / 60
  segundos %= 60
  
  fmt.Printf("%d:%d:%d\n", horas, minutos, segundos)
  
}
